src/sections/post-event/post-event.js

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+
+import style from "./post-event.module.scss"
+
+import Icon from "../../components/icon"
+import Spacer from "../../components/spacer"
+import TextBlock from "../../components/text-block"
+import logo from "../../assets/images/logo.svg"
+import Typography from "../../components/typography"
+
+const PostEvent = () => (
+  <div className={style.root}>
+    <Spacer size="xs" />
+    <TextBlock headline="!🎉 Thank you all for #jscc22 🎉!" />
+    <div className={style.logo}>
+      <Icon source={logo} />
+    </div>
+    <div className={style.text}>
+      <Typography variant="subline" paragraph>
+        Thanks to everyone who contributed and participated.
+        <br />
+        You are what makes this Bar Camp great!
+        <Spacer size="xs" />
+        Stay in tune, for when{" "}
+        <a href="https://twitter.com/hashtag/jscc22">#jscc22</a> becomes{" "}
+        <a href="https://twitter.com/hashtag/jscc23">#jscc23</a>
```

Comment:
Yea, looking forward to next year already 🤩 !

I still would suggest we add the Twitter and GitHub Links+Icons as we had before:
<img width="559" alt="image" src="https://user-images.githubusercontent.com/217931/185757602-2903c6b4-45f7-4b87-ac94-be47fdac0c1d.png">

But that can be done afterwards.